code/modules/recycling/disposal.dm

```diff
@@ -103,32 +115,55 @@
 			return
 		if((S.allow_quick_empty || S.allow_quick_gather) && length(S.contents))
 			S.hide_from(user)
-			user.visible_message("[user] empties \the [S] into \the [src].", "You empty \the [S] into \the [src].")
+			user.visible_message(
+				"<span class='notice'>[user] empties \the [S] into [src].</span>",
+				"<span class='notice'>You empty \the [S] into [src].</span>",
+				"<span class='notice'>You hear someone emptying something full of smaller objects out into \a [src].</span>"
```

Comment:
You shouldn't use `[src]` here because it'll default to the name, which can be changed